KafkaSourceFactory unflinkization

arkadius opened the pull request.

mproch commented on engine/flink/kafka-util/src/main/resources/META-INF/services/pl.touk.nussknacker.engine.util.json.ToJsonEncoder:
```diff
@@ -1 +1 @@
-pl.touk.nussknacker.engine.kafka.source.flink.InputMetaToJson
```
Let's move also this file to generic kafka-util

mproch reviewed (commented):
:+1: cool, looks good, added only one minor remark + I think we'll need changelog/migration guide

arkadius commented:
> +1 cool, looks good, added only one minor remark + I think we'll need changelog/migration guide

Added something ... I hope that it will help someone
